provider/qemu_img_utils.py

```diff
@@ -113,7 +113,7 @@ def strace(image, trace_events=None, output_file=None):
     image_cmd = image.image_cmd
     strace_prefix = ["strace"]
     if trace_events:
-        strace_prefix.extend(("-e", ",".join(trace_events)))
```

Comment:
How could you just simply add `-f` to the `strace` command without adding an extra parameter to the function as a switch?

Reply:
`-f` will trace the child processes, too. No need extra parameter. `man strace` will get more details. Thanks.

Reply:
LINE 116 is following the condition there are trace events specified. Appending `-f` here is misleading.

Reply:
I wrote this function in this module to provide general `strace` functionality to users.  Please do not limit its using scenario only based on your circumstances. I believe adding an extra parameter to the function as a switch to open `-f` is the best solution without any doubt.